### PR TITLE
Provide UA test vectors for the first 3 valid diversifier indices for each account

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

**Review by pacu — APPROVED** [review]
utACK

String-encoded UAs would be useful for testing the SDKs. Probably that change is better on another PR

**Comment by pacu** [comment]
CI is failing :/

File: .github/workflows/test_vectors.yml [hunk 1]
```diff
@@ -24,6 +24,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Install gnome-keyring
+        run: sudo apt-get install gnome-keyring
```
Comment (str4d) [inline .github/workflows/test_vectors.yml:28]: What is this needed for?

**Review by str4d — APPROVED** [review]
utACK 73ced0f5c986d7dceeb231ce5bf1822bb6dc2673

File: .github/workflows/test_vectors.yml (hunk 1 above)
Comment (nuttycom) [reply .github/workflows/test_vectors.yml:28]: One of the python dependencies refuses to build without it; that was the previous build failure.

**Pull request merged**